packages/react/src/generators/application/schema.json

```diff
@@ -165,11 +165,6 @@
       "enum": ["babel", "swc"],
       "default": "babel"
     },
-    "skipDefaultProject": {
```

Comment:
Keep it as a flag so commands don't fail but add `x-deprecated`.

Reply:
Fixed